tutorials/.CONTRIBUTING_TUTORIALS.md

```diff
@@ -0,0 +1,446 @@
+- Order tutorials logically by difficulty or dependency
+- Always include `index.md` as the first entry
+- Use folder names (without quotes) for subcategories
+
+## Writing Style and Format
+
+### Tutorial Structure Template
+
+```markdown
+# Tutorial Title
+
+## Introduction
+
+2-3 paragraphs explaining:
+- What the tutorial covers
+- Why it's important/useful
+- What users will learn/build
+
+## Prerequisites
+
+List required:
+- Prior knowledge
+- Tools and dependencies
+- Accounts and tokens needed
+- System requirements
+
+## Step 1: Setup and Installation
+
+Clear step-by-step instructions with:
+- Commands to run
+- Expected outputs
+- Troubleshooting tips
+
+## Step 2: [Main Content]
+
+Continue with logical progression...
+
+## Verification/Testing
+
+How to confirm the tutorial worked:
+- Expected results
+- Common issues and solutions
+
+## Where to Go Next
+
+Suggest related tutorials or next steps
+
+## Additional Resources
+
+Links to relevant documentation, tools, or communities
+```
+
+### Writing Best Practices
+
+1. **Use active voice**: "You will build..." instead of "A blockchain will be built..."
+2. **Be specific**: Provide exact commands, file paths, and parameter values
+3. **Include context**: Explain why each step is necessary
+4. **Use consistent terminology**: Follow established Polkadot vocabulary
+5. **Provide alternatives**: When multiple approaches exist, mention them
+6. **Add warnings**: Use `!!!warning` for important caveats
+
+### Formatting Guidelines
+
+- Use `bash` code blocks for terminal commands
+- Use appropriate language tags for code examples (`rust`, `javascript`, `solidity`)
+- Use numbered lists for step-by-step procedures
+- Use bullet points for feature lists or options
+- Use `!!!tip`, `!!!note`, and `!!!warning` for callouts
+
+## Code Examples and Snippets
+
+### Code Snippet Organization
+
+Store reusable code in the `.snippets` directory:
+
+```
+.snippets/code/tutorials/[category]/[subcategory]/[tutorial-name]/
+├── setup.sh
+├── example-contract.sol
+├── interaction-script.js
+└── expected-output.html
+```
+
+### Including Code Snippets
+
+```markdown
+```bash
+--8<-- 'code/tutorials/polkadot-sdk/parachains/setup.sh'
+```
+```
+
+### Code Block Best Practices
+
+- Always specify the language for syntax highlighting
+- Include file names when showing file contents
+- Use `!!!tip` to explain complex code sections
+- Test all code examples before submission
+- Include expected outputs where relevant
+
+## Images and Screenshots
+
+### Image Requirements
+
+- **Format**: All images must be in `.webp` format
+- **Location**: Store in `images/tutorials/[category]/[subcategory]/[tutorial-name]/`
+- **Naming**: Use descriptive names (e.g., `setup-wallet-connection.webp`)
+- **Alt text**: Always include descriptive alt text
+
+### Screenshot Guidelines
+
+- Use consistent browser/interface settings
+- Crop images to show only relevant content
+- Use arrows or highlighting to guide attention
```

Comment:
We have a template for arrows that tutorials and guides should adhere too. 
If the contributor uses their own arrows on the screenshots/images we have to re-take the screenshot or create a new image then add our arrows. This can be cumbersome.

@nhussein11 WDYT, should we add our arrows as a resource here?